dask-xfails.txt

```diff
@@ -37,6 +37,9 @@ array_api_tests/test_data_type_functions.py::test_finfo[float32]
 # (I think the test is not forcing the op to be computed?)
 array_api_tests/test_creation_functions.py::test_linspace
 
+# out.shape=(2,) but should be (1,)
```

Comment:
Is there an upstream issue for this?

Reply:
Not yet - but I'll followup with dask

(I haven't been able to get this down to a MRE yet)